bin/ch/262.js

```diff
@@ -17,7 +18,8 @@ var $262 = {
           agent:{
             receiveBroadcast: function(callback){ WScript.ReceiveBroadcast(callback); },
             report: function(value){ WScript.Report(value); },
-            leaving: function(){ WScript.Leaving(); }
+            leaving: function(){ WScript.Leaving(); },
+            monotonicNow: () => WScript.monotonicNow()
```

Comment:
Can we use a consistent style for these functions to be neater - I think we can just assign them:
```js
agent:{
  receiveBroadcast : WScript.ReceiveBroadcast,
  report : WScript.Report,
  leaving : WScript.Leaving,
  monotonicNow : WScript.monotonicNow
  }
  ```